spec/policy/rate_limit/rate_limit_spec.lua

```diff
@@ -62,207 +65,209 @@ describe('Rate limit policy', function()
   end)
 
   describe('.access', function()
-    it('success with multiple limiters', function()
-      local config = {
-        connection_limiters = {
-          { key = { name = 'test1', scope = 'global' }, conn = 20, burst = 10, delay = 0.5 }
-        },
-        leaky_bucket_limiters = {
-          { key = { name = 'test2', scope = 'global' }, rate = 18, burst = 9 }
-        },
-        fixed_window_limiters = {
-          { key = { name = 'test3', scope = 'global' }, count = 10, window = 10 }
-        },
-        redis_url = 'redis://'..redis_host..':'..redis_port..'/1'
-      }
-      local rate_limit_policy = RateLimitPolicy.new(config)
-      rate_limit_policy:access(context)
-    end)
+    describe('using #shmem', function()
+      setup(function()
+        ngx.shared.limiter = shdict()
+      end)
 
-    it('no redis url', function()
-      local config = {
-        connection_limiters = {
-          { key = { name = 'test1', scope = 'global' }, conn = 20, burst = 10, delay = 0.5 }
-        },
-        leaky_bucket_limiters = {
-          { key = { name = 'test2', scope = 'global' }, rate = 18, burst = 9 }
-        },
-        fixed_window_limiters = {
-          { key = { name = 'test3', scope = 'global' }, count = 10, window = 10 }
-        }
-      }
-      local rate_limit_policy = RateLimitPolicy.new(config)
-      rate_limit_policy:access(context)
-    end)
+      it('works with multiple limiters', function()
+        local rate_limit_policy = RateLimitPolicy.new({
+          connection_limiters = {
+            { key = { name = 'test1', scope = 'global' }, conn = 20, burst = 10, delay = 0.5 }
```

Comment:
The title says that  `it works with multiple limiters', but there's only one.

Reply:
Indeed. Will fix.

Reply:
Fixed. Rebased.